src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574666,15 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040002
+name: PIK3CA-related overgrowth spectrum
+def: "Somatic PIK3CA mutation that has a congenital or early childhood onset. Phenotypic overgrowth presentation with a mixture of other features including vascular malformations, epidermal nevus, large isolated lymphatic malformations, isolated macrodactyly, overgrown splayed feet/hands, overgrown limbs, truncal adipose overgrowth, hemimegalencephaly/dysplastic megalencephaly/focal cortical dysplasia, seborrheic keratoses, and or benign lichenoid keratoses." [https://clinicalgenome.org/affiliation/50020/, https://orcid.org/0000-0002-0587-4693, PMID:23592320, PMID:23946963, PMID:25557259]
+synonym: "MONDO:0100283" BROAD [https://clinicalgenome.org/affiliation/50020/, PMID:23592320, PMID:25557259]
```

Comment:
the synonym should be a string/name. (this is probably a copy/paste error).

Reply:
Comment addressed.